libs/blocks/preflight/preflight.css

```diff
@@ -226,6 +226,12 @@ p.preflight-date-wrapper .disabled-publish {
 
 .preflight-group-row a {
```

Comment:
This will target more links than intended, including the one that leads to the source document. How about you apply the changes more selectively to the relevant link?